llvm/lib/Target/RISCV/RISCVVLOptimizer.cpp

```diff
@@ -178,6 +178,19 @@ static unsigned getIntegerExtensionOperandEEW(unsigned Factor,
   return Log2EEW;
 }
 
+#define VSEG_CASES(Prefix, EEW)                                                \
```

Comment:
Instead of using macros, we can also use the search tables we have for segmented loads/stores here. I'm open to either way